pkg/rados/config.go

```diff
@@ -1,10 +1,20 @@
 package rados
 
-import "time"
+import (
+	"time"
+)
 
 type Config struct {
 	User         string        `yaml:"user"`
 	UserKeyring  string        `yaml:"userKeyring"`
 	MonHost      string        `yaml:"monHost"`
 	RadosTimeout time.Duration `yaml:"radosTimeout"`
+	UseMock      bool          `yaml:"useMock"`
```

Comment:
do we need this config if we have mock build flag? what will happen if `UseMock=true` and build flag is not provided?